src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { Collapse, styled } from '@mui/material';
+
+import { theme, variables, StyledFlexTopCenter } from 'shared/styles';
+
+export const StyledSliderPanelContainer = styled(Collapse)`
```

Comment:
Maybe you can use StyledItemOptionContainer, but rewrite a padding value from it?

Reply:
> Maybe you can use StyledItemOptionContainer, but rewrite a padding value from it?

The main thing is in using Mui Collapse as a base component. StyledItemOptionContainer uses StyledFlexColumn